downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -84,6 +84,126 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 - Introduced GraphQL server-side caching support
 
+#### `data.jsondata` package
+
+The [`data.jsondata`](https://lib.ballerina.io/ballerina/data.jsondata/latest/) package has been introduced to provide a set of APIs for JSON data conversions, data projection, and JSON navigation.
```

Comment:
Sorry, missed this previously, but shall we use functions instead of APIs?

Reply:
Updated https://github.com/ballerina-platform/ballerina-dev-website/pull/8877/commits/4b7f2abbd9a0299e5ce59414cb437488f4d2a2e5